src/metabase/config.clj

```diff
@@ -101,6 +102,13 @@
    Looks something like `Metabase v0.25.0.RC1`."
   (str "Metabase " (mb-version-info :tag)))
 
+;; This variable used to live at `metabase.metabot.instance/local-process-uuid`
+(defonce ^{:doc "This UUID is randomly-generated upon launch and used to identify this specific Metabase instance during this specifc
+                run. Restarting the server will change this UUID, and each server in a hortizontal cluster will have its own ID,
+                making this different from the `site-uuid` Setting. The local process UUID is used to differentiate different
+                horizontally clustered MB instances so we can determine which of them will handle MetaBot duties."}
+  local-process-uuid (str (UUID/randomUUID)))
```

Comment:
A couple other style things:

*  Since this is a multiline `def` form you should put `(str (UUID/randomUUID))` on its own line
*  Our general column width goal is 120 chars, because that's what GitHub shows without cutting it off. Tweak your editor to do that for you. If you want to add a config file for your editor to the repo feel free to do so, we already have one that has those sorts of settings for Emacs. 
*  I don't think it's worth putting a comment about where this variable used to live, hopefully everyone can forget it ever lived there moving forward. It's not info that affects how you'd use it so let's not clutter people's heads with that info, you can always go in Git history if you want to see the history of the code base

Reply:
Thanks for that! :+1: